tasks/internal_reconcile_report_mismatch/schemas/output.json

```diff
@@ -1,91 +1,95 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_mismatch/schemas/output.json",
-    "title": "internal_reconcile_report_mismatch Lambda Output",
-    "description": "The output for the internal_reconcile_report_mismatch Lambda.",
-    "type": "object",
-    "required": [
-      "jobId",
-      "anotherPage",
-      "mismatches"
-    ],
-    "properties": {
-        "jobId": {
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_mismatch/schemas/output.json",
+  "title": "internal_reconcile_report_mismatch Lambda Output",
+  "description": "The output for the internal_reconcile_report_mismatch Lambda.",
+  "type": "object",
+  "required": [
+    "jobId",
+    "anotherPage",
+    "mismatches"
+  ],
+  "properties": {
+    "jobId": {
+      "type": "integer",
+      "description": "The unique ID of the reconciliation job."
+    },
+    "anotherPage": {
+      "description": "Indicates if more results can be retrieved on another page.",
+      "type": "boolean"
+    },
+    "mismatches": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "collectionId",
+          "granuleId",
+          "filename",
+          "keyPath",
+          "cumulusArchiveLocation",
+          "orcaEtag",
+          "s3Etag",
+          "orcaLastUpdate",
+          "s3LastUpdate",
+          "orcaSizeInBytes",
+          "s3SizeInBytes",
+          "discrepancyType",
+          "comment"
```

Comment:
some of these might become optional after https://bugs.earthdata.nasa.gov/browse/ORCA-412